package.json

```diff
@@ -50,13 +52,31 @@
     "docs:watch": "gitbook install && gitbook serve",
     "prepublish": "nwb build",
     "release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci",
-    "test:ts": "tsfmt -r && tsc -p ./",
-    "test:watch": "nwb test --server"
+    "pretest": "npm run style && npm run test:node && npm run test:ts",
```

Comment:
Maybe if I had more background in how all of them are tied together and used. It just feels like there's a lot of complexity now for a 3k lib.